setup.py

```diff
@@ -31,7 +31,9 @@
     ],
     include_package_data=True,
     install_requires=[
-        'Django>=1.11,<2.1',
```

Comment:
can't we sum up the 2nd and 3rd line into one as they result in the same python version?

Reply:
The 3rd line should actually read “>3.4” as Django supports python 3.4 only up to Django 2.0. Django 2.1 supports python 3.5 and newer only https://docs.djangoproject.com/en/2.1/releases/2.1/